src/tests/electron/common/view-controllers/app-controller.ts

```diff
@@ -125,4 +127,36 @@ export class AppController {
             },
         );
     }
+
+    public getServerLog(logName: string, extraLogNames: string): string {
+        return fs
+            .readFileSync(
+                path.normalize(
+                    `drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/server.log`,
+                ),
+            )
+            .toString();
+    }
+
+    public getAdbLog(logName: string, extraLogNames: string): string {
+        return fs
+            .readFileSync(
+                path.normalize(`drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/adb.log`),
+            )
+            .toString();
+    }
+
+    public resetServerLog(logName: string, extraLogNames: string): void {
+        fs.rmdirSync(
+            path.normalize(`drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/server.log`),
+            { recursive: true },
+        );
+    }
+
+    public resetAdbLog(logName: string, extraLogNames: string): void {
+        fs.rmdirSync(
```

Comment:
I *think* that adb.log is a file, but we're use `fs.rmdirSync` to remove it. Should we be using `fs.unlink`, instead?

Reply:
Yes! Now using `fs.unlinkSync`